File: LMarksBookmarksApi/src/LMarksBookmarksApi/Models/Bookmark.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LMarksBookmarksApi.Models
{
    public class Bookmark
    {
        public string Key { get; set; }
        public string User { get; set; }
        public string Link { get; set; }
        public string Description { get; set; }
        public DateTime Date { get; set; }
    }
}

File: LMarksWebClient/src/app/bookmark.ts
export class Bookmark{
    key : string;
    user : string;
    link : string;
    description : string;
    date : Date;
    visible : number;

    constructor(user:string, link:string, description:string){
        this.user = user;
        this.link = link;
        this.description = description;
  }
}